verifier/src/jwt-utils.js

```diff
@@ -0,0 +1,28 @@
+/**
```

Comment:
Why don't we use a jwt library, as in the app?

Reply:
Oh! I thought that at the moment I only need to decode it, and it is fairly simple without external libraries. When I need to check signatures I'll add it for sure 😄 !